Directory.Build.props

```diff
@@ -51,8 +51,8 @@
     <PackageReference Include="BepInEx.Core" Version="5.*" PrivateAssets="all" />
     <PackageReference Include="UnityEngine.Modules" Version="2018.4.12" IncludeAssets="compile" PrivateAssets="all" />
     <PackageReference Include="DysonSphereProgram.GameLibs" Version="*-*" IncludeAssets="compile" PrivateAssets="all" />
-     <PackageReference Include="DysonSphereProgram.Modding.CommonAPI" Version="*-*" IncludeAssets="compile" PrivateAssets="all"
```

Comment:
I'm glad you noticed this, I was just gonna ninja fix it haha, didn't want to hassle you 😂

Reply:
I need to figure out why my editor is so intent of 4 spaces for these files because it's a pain to make it use 2